File: app/src/main/java/app/android/gifloaderviewmodel/controller/MyApplication.java
package app.android.gifloaderviewmodel.controller;

import android.app.Application;
import android.widget.Toast;

import io.reactivex.functions.Consumer;
import io.reactivex.plugins.RxJavaPlugins;

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        RxJavaPlugins.setErrorHandler(throwable -> {
            throwable.printStackTrace();
            Toast.makeText(getBaseContext(), "Error", Toast.LENGTH_SHORT).show();
        });
    }
}

File: app/src/main/java/app/android/gifloaderviewmodel/controller/MyApplication.kt
package app.android.gifloaderviewmodel.controller

import android.app.Application
import android.widget.Toast
import io.reactivex.plugins.RxJavaPlugins

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()

        RxJavaPlugins.setErrorHandler(fun(throwable: Throwable) {
            throwable.printStackTrace()
            Toast.makeText(baseContext, "Error", Toast.LENGTH_SHORT).show()
        })
    }
}